test/e2e/framework.go

```diff
@@ -1046,6 +1046,22 @@ func (data *TestData) createNginxPodOnNode(name string, ns string, nodeName stri
 	}, hostNetwork, nil)
 }
 
+// createHostNetworkNginxPodOnNode creates a hostNetwork Pod compared with createNginxPodOnNode.
+func (data *TestData) createHostNetworkNginxPodOnNode(name string, ns string, nodeName string) error {
```

Comment:
No need to add this. See https://github.com/antrea-io/antrea/blob/b0e229982ee96a3d93f7c5a1b563291b36af1d47/test/e2e/framework.go#L1039

Reply:
Removed.